handler/misc.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Prometheus Authors
```

Comment:
It's legally (almost) irrelevant, but I'd put the current year into newly created files.

Reply:
done